tests/boardfarm_plugins/boardfarm_prplmesh/devices/prplmesh_docker.py

```diff
@@ -5,12 +5,12 @@
 
 import logging
 import os
-import random
 import sys
 import time
 
 from environment import ALEntityDocker
 from .prplmesh_base import PrplMeshBase
+from utils import get_rand_mac
```

Comment:
I suggest to add a class with static functions, then you can just import the class when using one of the functions instead of importing each function on its own.

Reply:
> I suggest to add a class with static functions, then you can just import the class when using one of the functions instead of importing each function on its own.

Indeed.
But as mentioned @arnout in his [comment](https://github.com/prplfoundation/prplMesh/pull/1136#discussion_r408968971) to #1136 draft PR - that function will become useless soon, so I'll remove it here at all.